src/store/filesystem.rs

```diff
@@ -1,80 +1,30 @@
 //! A filesystem-backed Zarr hierarchy.
 
-use std::fs::{
-    self,
-    File,
-};
-use std::io::{
-    BufReader,
-    BufWriter,
-    Error,
-    ErrorKind,
-    Result,
-};
-use std::path::PathBuf;
+use std::fs::{self, File};
+use std::io::{BufReader, BufWriter, Error, ErrorKind, Result};
+use std::path::{Path, PathBuf};
 
 use fs2::FileExt;
-use serde_json::{
-    self,
-    json,
-    Value,
-};
 use walkdir::WalkDir;
 
 use crate::{
-    storage::{
-        ListableStore,
```

Comment:
Perhaps just delete this file and crate-level depedencies (eg tests and benches) and see if it compiles. We could strip down this repo to the bare minimum. Perhaps then we could simply merge the stripped down files directly into private repo